### PR TITLE
Feature/saa 417 enables job to deallocate offenders

### PR DESCRIPTION
**Pull request by matt-crowson-moj**
Includes change to fix content type issue seen here on all jobs:

image

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsactivitiesmanagementapi/resource/JobTriggerController.kt [hunk 1]
```diff
@@ -16,7 +16,7 @@ import uk.gov.justice.digital.hmpps.hmppsactivitiesmanagementapi.job.OffenderDea
 // within the namespace without requiring authentication
 
 @RestController
-@RequestMapping("/job", produces = [MediaType.APPLICATION_JSON_VALUE])
+@RequestMapping("/job", produces = [MediaType.TEXT_PLAIN_VALUE])
```
Comment (matt-crowson-moj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsactivitiesmanagementapi/resource/JobTriggerController.kt:19]: Triggers are returning plain text.

**Review by timharrison-moj — COMMENTED** [review]
One question, but LGTM.

File: helm_deploy/hmpps-activities-management-api/templates/deallocate-offenders-cronjob.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  jobTemplate:
+    spec:
+      backoffLimit: 2
+      template:
+        spec:
+          restartPolicy: Never
+          activeDeadlineSeconds: 3600
+          containers:
+          - name: hmpps-activities-management-api
+            image: ghcr.io/ministryofjustice/hmpps-devops-tools
```
[the remark this answers is not in the document]
Comment (matt-crowson-moj) [reply helm_deploy/hmpps-activities-management-api/templates/deallocate-offenders-cronjob.yaml:20]: Done

**Review by timharrison-moj — APPROVED** [review]
LGTM

**Pull request merged**